tokio/docs/reactor-refactor.md

```diff
@@ -188,7 +188,7 @@ readiness, the driver's tick is packed into the atomic `usize`.
 The `ScheduledIo` readiness `AtomicUsize` is structured as:
 
 ```
-| reserved | generation |  driver tick | readinesss |
+| reserved | generation |  driver tick | readiness |
```

Comment:
I can add the extra space (or remove one from the rows below). Technically markdown doesn't really care.

I'm not sure offhand if it'd be added before or after the word. The preceding column has the extra character before...

Reply:
it might be nice to just resize the rest of the table to match, but, yeah, markdown doesn't care, so i don't think this is a big deal regardless.